src/ivc/cyclefold/support_circuit/mod.rs

```diff
@@ -144,15 +177,22 @@ mod tests {
 
         let tmp = p0 * l0;
         trace!("p0 * l0_bits = {:?}", tmp);
-        let p2 = (p0 * l0) + (p1 * l1);
+        let p_out = ((p0 * l0) + (p1 * l1)).into();
```

Comment:
Why? This is folding of W_commitment[9] 
One accumulation is:
`acc[9] * l0 + incoming[9] * l1` and same for every `W_commitment`

Reply:
https://github.com/snarkify/sirius/blob/f626c785c24e86e9a294f27747e67922d2722049/src/nifs/protogalaxy/mod.rs#L176

https://github.com/snarkify/sirius/blob/f626c785c24e86e9a294f27747e67922d2722049/src/nifs/protogalaxy/mod.rs#L199

Reply:
I just started with a simple variant where we immediately have new_acc.W_commitment value. Otherwise, we would need to do sum two points via `bn` logic. This is probably more optimal, just longer to impl